flake.nix

```diff
@@ -2,7 +2,7 @@
   description = "basic-webserver devShell flake";
 
   inputs = {
-    roc.url = "github:roc-lang/roc";
+    roc.url = "github:smores56/roc?ref=remove-task";
```

Comment:
This can now be reverted since `snake_case` builtins were merged

Reply:
Fixed in d04f1a9